Lib/getpass.py

```diff
@@ -132,14 +132,15 @@ def win_getpass(prompt='Password: ', stream=None, *, echo_char=None):
     return pw
 
 
-def fallback_getpass(prompt='Password: ', stream=None):
+def fallback_getpass(prompt='Password: ', stream=None, *, echo_char=None):
+    _check_echo_char(echo_char)
     import warnings
     warnings.warn("Can not control echo on the terminal.", GetPassWarning,
                   stacklevel=2)
     if not stream:
         stream = sys.stderr
     print("Warning: Password input may be echoed.", file=stream)
-    return _raw_input(prompt, stream)
+    return _raw_input(prompt, stream, echo_char=echo_char)
```

Comment:
Can you test manually that passing an echo_char here works as `fallback_getpass` directly writes to the stream using `readline`? for instance, if the stream is a simple `StringIO()`, then we would write more stuff than necessary (e.g., `\b` wouldn't be understood correctly).

Reply:
if I understood you correctly, my test:
```python
>>> fallback_getpass(stream=s, echo_char='*')
<python-input-5>:1: GetPassWarning: Can not control echo on the terminal.
passw0rd! # I erased chars here few times
'passw0rd!'
>>> s.getvalue()
Warning: Password input may be echoed.
Password: ********* # it's contain only `passw0rd!` without `\b` chars
```

Reply:
Ah! actually, there won't be any dynamic feedback. Ok nvm then.